src/lib/services/client-metrics/client-metrics.test.ts

```diff
@@ -318,27 +361,27 @@ test('should have correct values for lastHour', () => {
     let c = metrics.getTogglesMetrics();
     expect(c.lastHour.toggle).toEqual({ yes: 41, no: 41 });
 
-    jest.advanceTimersByTime(10 * 1000);
+    jest.advanceTimersByTime(10_000);
     c = metrics.getTogglesMetrics();
     expect(c.lastHour.toggle).toEqual({ yes: 41, no: 41 });
 
     // at 30
-    jest.advanceTimersByTime(30 * 60 * 1000);
+    jest.advanceTimersByTime(minutesToMilliseconds(30));
```

Comment:
… but when we're dealing with minutes, the numbers are less easy to grok, so I prefer the helper functions over either a constant number of milliseconds or inline arithmetic